admin-dev/themes/new-theme/js/pages/product/combination/bulk-form-handler.ts

```diff
@@ -190,7 +193,11 @@ export default class BulkFormHandler {
       // @todo when the ProgressModal will be integrated this will update it after each request
       try {
         // eslint-disable-next-line no-await-in-loop
-        const response: Response = await this.combinationsService.bulkUpdate(Number(checkbox.value), new FormData(form));
+        const response: Response = await this.combinationsService.bulkUpdate(
+          Number(this.productId),
```

Comment:
This one is redundant since `this.productId` is already a number, but not critical anyway

Reply:
I was talking about the casting to Number